hawk/app/assets/stylesheets/shared/_table.scss

```diff
@@ -31,5 +31,11 @@
         }
       }
     }
+    td.col-sm-2 {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3